resources/codegen-composer.js

```diff
@@ -296,9 +306,13 @@
       }
 
       if (html) {
-        fullDescription += obj.possibleValues.map((val) => `<code>${val}</code>`).join(", ");
+        fullDescription += obj.possibleValues.map((val) => 
```

Comment:
Right now it produces:
![obraz](https://github.com/user-attachments/assets/a2c3a16d-7155-4730-a352-3a4559bc7227)
that is quite unreadable 😅

Reply:
Added "Show more" button
![CleanShot 2024-12-05 at 20 27 30](https://github.com/user-attachments/assets/fc83abfa-2948-4a09-be37-0bccb56b0747)